packages/sdk/connect/src/utils/formatter.ts

```diff
@@ -45,7 +45,7 @@ export function inputCeloTxFormatter(tx: CeloTx): FormattedCeloTx {
   formattedTX.from = inputAddressFormatter(from?.toString())
   formattedTX.to = inputAddressFormatter(to)
 
-  formattedTX.gas = numberToHex(gas)
+  formattedTX.gas = numberToHex(gas?.toString())
```

Comment:
calling toString on big numbers returns scientific notation. use toFixed()

Reply:
actually we should look into why numberToHex needs a string

Reply:
i reverted the change but I agree, maybe another PR though